diplom.py

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/python
-print "hello"
+import ml_metrics as metrics
+import argparse
+from sklearn import linear_model
+from sklearn.ensemble import RandomForestClassifier
+
+parser = argparse.ArgumentParser()
```

Comment:
Final thing - lets move this parsing after function definition so the execution flow would not be interrupted by function definitions.